src/store/atoms/atomWithStorage.ts

```diff
@@ -0,0 +1,11 @@
+import { atomWithStorage } from 'jotai/utils';
+
+export const selectedPhotoAtomWithStorage = atomWithStorage<string>(
```

Comment:
저도 토큰을 sessionStorage에 저장할때 이 함수를 사용하려고 했는데, 버그가 너무 많이 발생했다는 인터넷 글을 봐서 사용하지 않았습니다. 
수정사항을 제안해 볼테니 괜찮으면 반영해 주시고 복잡하면 반영 안해주셔도 좋습니다.

```suggestion
// @/store/atoms/tokenActions.ts
export const actionInsertToken = atom(
  (get) => get(tokenAtom),
  (_, set, token: string) => {
    set(tokenAtom, token);
    window.sessionStorage.setItem('accessToken', token);
  },
);
```

Reply:
자료가 별로 없긴 하더라고요
근데 `localStorage`말고 `sessionStorage` 쓰신 이유가 궁금합니다!

Reply:
저는 로그인 기록이 사용자 컴퓨터에 저장되어 있는 상태를 별로 좋게 생각하지 않아서 세션이 끝나면 세로 로그인하게 만들었습니다.  
사용자 입장에서는 로그인하는 과정이 귀찮을 수 있지만 보안이라는 관점으로 넘어가면 이는 사용자 경험과 타협할 수 없다고 생각합니다. 사용자 경험보다는 사용자 정보가 더 중요하니까요

Reply:
넵 답변 감사합니다!